day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
Nice job on Fizzbuzz! I'm happy to see that you are accepting user input - pretty cool! If you have time to attempt other day-7 challenges then I encourage you to do so. They will help build up those ruby muscles! You can time box these to ensure you don't spend too much time working on them, and get some practice with a time constraint, which you will see a lot at Turing. Great job on this capstone Graham!